src/seedu/addressbook/ui/TextUi.java

```diff
@@ -116,6 +117,14 @@ public void showToUser(String... message) {
             out.println(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX));
         }
     }
+    
+    public String getPrintableString(Printable... printables) {
+    	String result = "";
+    	for (Printable printable: printables) {
+    		result = result + printable.getPrintableString();
+    	}
+    	return result;
+    }
 
```

Comment:
A good try. What to do you think if this goes in ReadOnlyPerson interface? I think it looks more meaningful in Person context.